client/my-sites/plugins/marketplace/marketplace-domain-upsell/index.tsx

```diff
@@ -142,6 +143,7 @@ function CalypsoWrappedMarketplaceDomainUpsell(): JSX.Element {
 			( { product_slug: added_product_slug } ) => added_product_slug === product_slug
 		);
 		setDomainProductUUID( productAdded?.uuid ?? '' );
+		dispatch( setPrimaryDomain( suggestion?.domain_name ) );
```

Comment:
I assume we intentionally prefer setting the primary domain to `undefined` (rather than `null` or an empty string for example) in the instance when it's not present in the suggestion? Just wanted to confirm it's intentional.

Reply:
Yeah this never supposed to be an empty value, but for some reason the DomainSuggestion type has this value as optional. So we tolerate undefined in case the value becomes undefined. No strong reason to make it null forcefully.